mmv1/third_party/terraform/services/bigquery/resource_bigquery_table_internal_test.go

```diff
@@ -550,14 +570,18 @@ var testUnitBigQueryDataTableIsChangableTestCases = []testUnitBigQueryDataTableJ
 
 func TestUnitBigQueryDataTable_schemaIsChangable(t *testing.T) {
```

Comment:
Do I understand correctly that this test is testing two different things - isChangeable when the schema is at the top level and when it's nested? Can we split these two cases?

Also, not related to your change but "Changable" should be "Changeable".

Reply:
Done